client/post-editor/editor-tags/index.jsx

```diff
@@ -63,28 +63,27 @@ export default React.createClass( {
 	},
 
 	render: function() {
-		var tagNames = ( this.props.tags || [] ).map( function( tag ) {
+		const tagNames = ( this.props.tags || [] ).map( function( tag ) {
 			return tag.name;
 		} );
 
+		const helpText = isPage( this.props.post )
+			? this.translate( 'Use tags to associate more specific keywords with your pages.' )
+			: this.translate( 'Use tags to associate more specific keywords with your posts.' );
+
 		return (
-			<label className="editor-drawer__label">
-				<span className="editor-drawer__label-text">
+			<div>
```

Comment:
>Does this change the effect of clicking the label?

Yeah, unfortunately it looks like this is a regression, though we're also not very consistent in how we take advantage of this behavior. It'd be a nice accessibility boost if we were to, though.

Fix for this might be to change semantics to render `<EditorDrawerLabel />` input children directly under the rendered `label`, and accept a separate prop for the `__label-text` element?

```jsx
<EditorDrawerLabel helpText={ helpText } label={ this.translate( 'Tags' ) }>
	<TokenField
		value={ this.getPostTags() }
  		displayTransform={ unescapeString }
  		suggestions={ tagNames }
  		onChange={ this.onTagsChange }
  		maxSuggestions={ TermsConstants.MAX_TAGS_SUGGESTIONS } />
</EditorDrawerLabel>
```

Or be even more flexible in detecting whether said `label` prop is passed and if not, treat the child as the `__label-text` content (would save updating a few files).